projects/storefrontlib/src/cms-components/cart/cart-shared/cart-item/cart-item.component.html

```diff
@@ -1,170 +1,219 @@
-<div [ngClass]="compact ? 'cx-compact row' : 'row'">
-  <!-- Item Image -->
-  <div class="col-2 cx-image-container">
-    <a
-      [routerLink]="{ cxRoute: 'product', params: item.product } | cxUrl"
-      (click)="viewItem()"
-      tabindex="-1"
-    >
-      <cx-media [container]="item.product.images?.PRIMARY"></cx-media>
-    </a>
-  </div>
-  <!-- Item Information -->
-  <div class="cx-info col-10">
-    <div class="cx-info-container row">
-      <!-- Item Description -->
-      <div [ngClass]="compact ? '' : ' col-md-3 col-lg-3 col-xl-5'">
-        <div *ngIf="item.product.name" class="cx-name">
-          <a
-            class="cx-link"
-            [routerLink]="{ cxRoute: 'product', params: item.product } | cxUrl"
-            (click)="viewItem()"
-            >{{ item.product.name }}</a
-          >
-        </div>
-        <div *ngIf="item.product.code" class="cx-code">
-          {{ 'cartItems.id' | cxTranslate }} {{ item.product.code }}
-        </div>
-        <!-- Generic Configuration-->
+<ng-container *ngIf="item.statusSummaryList[0]?.numberOfIssues > 0">
```

Comment:
This does not work for non-configurable items or items without status, cart display breaks
(Cannot read property '0' of undefined). We need to check for item.statusSummaryList !== undefined as well and should introduce a component function for that to not code this in the view

Reply:
We can add "**?**" after item like <ng-container *ngIf="item?.statusSummaryList[0]?.numberOfIssues > 0">

Reply:
This is not sufficient. I'm working on this.

Reply:
Done.